toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -215,9 +215,14 @@ Examples:
     cache store gems-$SEMAPHORE_GIT_BRANCH-revision-$(checksum Gemfile.lock) vendor/bundle
 
 Archives file or directory specified by path and associates it with key.
-Any further changes of path after the store operation completes will not
+As `cache store` uses `tar` leading `/` is automatically removed from the path.
+Also, any further changes of path after the store operation completes will not
 be automatically propagated to cache. The command always passes (exits with 0).
 
+Allocated disk space for cache is currently limited to 9.6G. 
+Also, in case of insufficient disk space, command `cache store` allocates space
+for new archive by deleting the oldest keys. 
+
```

Comment:
I would rephrase it as:

Also, in case of insufficient disk space, the `cache store` command frees disk space by deleting the
oldest keys.